requirements.txt

```diff
@@ -1,8 +1,9 @@
 django-notifications-hq~=1.6.0
 channels~=2.4.0
 # TODO set the next point release of openwisp-users when ready
-openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/master
-openwisp-utils[rest]
+openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/issues/253-register-menu-groups
+# TODO set the next point release of openwisp-utils when ready
+openwisp-utils[rest] @ https://github.com/niteshsinha17/openwisp-utils/tarball/gsoc/new-navigation-menu-design
```

Comment:
Before merge, please make it point to `openwisp/openwisp-utils` from `niteshsinha17/openwisp-utils`.